R/p_direction.R

```diff
@@ -16,6 +16,15 @@
 #'   \item It is strongly correlated with the frequentist p-value, and can thus be used to draw parallels and give some reference to readers non-familiar with Bayesian statistics.
 #' }
 #'
+#' \cr \cr
+#' \strong{Relationship with the p-value}
+#' \cr
+#' In most cases, it seems that the \emph{pd} chas a direct correspondance with the frequentist one-sided \emph{p}-value through the formula \ifelse{html}{\out{p<sub>one sided</sub> = 1-p<sub>d</sub>/100}}{\eqn{p_{one sided} = 1-\frac{p_{d}}{100}}} and to the two-sided p-value (the most commonly reported one) through the formula \ifelse{html}{\out{p<sub>two sided</sub> = 2*(1-p<sub>d</sub>/100)}}{\eqn{p_{two sided} = 2*(1-\frac{p_{d}}{100})}}. Thus, a two-sided p-value of respectively \code{.1}, \code{.05}, \code{.01} and \code{.001} would correspond approximately to a \code{pd} of \code{95\%}, \code{97.5\%}, \code{99.5\%} and \code{99.95\%}.
```

Comment:
@strengejacke It seems that this paragraph is making travis fails with the error: `! LaTeX Error: There's no line here to end.` do your raven's eyes manage to see what could possibly be wrong?

Reply:
hm, not really. debugging Latex errors in PDFs is a pain.